src/component/variantPage/BasicInfo.tsx

```diff
@@ -2,66 +2,111 @@ import * as React from 'react';
 import "./BasicInfo.css";
 import { observer } from "mobx-react";
 import { Row, Col } from "react-bootstrap";
+import { VariantAnnotationSummary } from 'cbioportal-frontend-commons';
+import _ from 'lodash';
 
 interface IBasicInfoProps
 {
-    // TODO: need to pass real data into this component
-    data?: string;
+    annotation: VariantAnnotationSummary | undefined;
 }
 
+export type BasicInfoData = {
+    name: string;
+    value: string | undefined;
+}
 @observer
 class BasicInfo extends React.Component<IBasicInfoProps>
 {
+    private parseAnnotation(annotation: VariantAnnotationSummary | undefined): BasicInfoData[] {
+        let parsedData:BasicInfoData[] = [];
+        if (annotation !== undefined) {
+            parsedData.push({
+                "name": "Allele",
+                "value": annotation.genomicLocation.referenceAllele + "/" + annotation.genomicLocation.variantAllele
```

Comment:
Use literals (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of string concatenation:

```suggestion
                "value": `${annotation.genomicLocation.referenceAllele}/${annotation.genomicLocation.variantAllele}`
```